ppr-ui/src/views/newMhrRegistration/SubmittingParty.vue

```diff
@@ -23,22 +23,55 @@
       </p>
 
       <!-- Insert Attention or Reference Number here -->
+      <v-card flat rounded id="attention-or-reference-number-card" class="mt-8 pa-8 pr-6 pb-3">
+        <v-row no-gutters class="pt-3">
+          <v-col cols="12" sm="2" >
+            <label class="generic-label" :class="{'error-text': false}">Attention or Reference Number</label>
+          </v-col>
+          <v-col cols="12" sm="10" class="px-1">
+            <v-text-field
+              filled
+              id="attention-or-reference-number"
+              class="pr-2"
+              label="Attention or Reference Number (Optional)"
+              v-model="attentionReferenceNum"
+              :rules="attentionReferenceNumRule"
```

Comment:
If we are only validating for the length (which looks like we are), we can use `maxLength` rule.

Reply:
yeah this is because the error text is different, in the design it says "Maximum 40 characters reached", but `maxLength()` can only return "Maximum 40 characters"

Reply:
Hmm, seems a bit weird to have unique messaging here vs everywhere else. 
Perhaps use our generic message. `Maximum 40 characters` and we push back a bit on the extra verbiage for consistency.

Reply:
updated using the compostable !